faculty_mill/publish.py

```diff
@@ -0,0 +1,94 @@
+import os
+from pathlib import Path
+from typing import Optional
+from uuid import UUID
+from time import sleep
+import click
```

Comment:
Add blank line between built-in and 3rd party libraries

Reply:
If we're going to adopt a style of keeping imports sorted (which we should!), let's setup `isort` so we can automate it and not need to think about it again.

Reply:
added a few pre-commit hooks for this exact purpose